### PR TITLE
shadow: Update to 4.6

### PR DESCRIPTION
**Pull request by neheb**
Switched dead URLs to new upstream.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @sbyx 
Compile tested: ipq806x

### CODE REVIEW

File: utils/shadow/Makefile [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
-PKG_RELEASE:=8
+PKG_VERSION:=4.6
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=http://pkg-shadow.alioth.debian.org/releases
-PKG_HASH:=3b0893d1476766868cd88920f4f1231c4795652aa407569faff802bcda0f3d41
+PKG_SOURCE_URL:=https://github.com/shadow-maint/shadow/releases/download/$(PKG_VERSION)
+PKG_HASH:=0998c8d84242a231ab0acb7f8613927ff5bcff095f8aa6b79478893a03f05583
+
```
[the remark this answers is not in the document]
Comment (neheb) [reply utils/shadow/Makefile]: I added the newline for consistency with other packages.

[the remark this answers is not in the document]
Comment (pprindeville) [reply utils/shadow/Makefile]: I would leave that to the package owner.

[the remark this answers is not in the document]
Comment (neheb) [reply utils/shadow/Makefile]: I don't think that's valid since he has not been active.

[the remark this answers is not in the document]
Comment (pprindeville) [reply utils/shadow/Makefile]: His name is still on this.  If you want to supplant him, that's a separate discussion and we can have that one. 

But in the meantime, if you want approval, please make that change.

Non-maintainer changes should always be "low touch".

**Comment by neheb** [comment]
Removed newline

**Comment by dibdot** [comment]
no feedback from maintainer - let's merge it.

**Pull request merged**